indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -657,6 +669,37 @@ public String getType()
     }
   }
 
+  private class HandoffTaskGroupNotice implements Notice
+  {
+    final Integer taskGroupId;
```

Comment:
```suggestion
    final List<Integer> taskGroupId;
```

I think accepting a List of integers in the notice will make for nicer dev ergonomics